packages/cli/src/commands/template/utils.ts

```diff
@@ -0,0 +1,118 @@
+import * as e2b from 'e2b'
```

Comment:
This file was moved from `/template/build.js` for readability and mocking in the added tests.